device_samples/getting_started/src/connections/proxy.js

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
I don't see a TypeScript version of this sample

Reply:
yeah the TS version still needs to be created. We had trouble getting it working a few months ago remember?